_layouts/home.html

```diff
@@ -15,12 +15,18 @@ <h2 class="post-list-heading">{{ page.list_title | default: "Posts" }}</h2>
       {%- for post in site.posts -%}
       <li>
         {%- assign date_format = site.minima.date_format | default: "%b %-d, %Y" -%}
+{%- assign author = site.data.authors[post.author] -%}
```

Comment:
Looks good! Please can you format the document so it's all indented correctly?

Reply:
Done!